layouts/_default/single.html

```
@@ -6,269 +6,45 @@
-            <a href="{{ printf `categories/%s/` ($category | urlize) | relURL }}" class="card-tag">{{- $category -}}</a>
-            {{- else -}}
-            {{- with .Params.category -}}<a href="{{ printf `categories/%s/` (. | urlize) | relURL }}"
-                class="card-tag">{{
-                . }}</a>{{- end -}}
+        {{- if .Params.hero -}}
+        {{- partial "product-hero.html" . -}}
+        {{- else -}}
+        <header class="article-header" style="margin-bottom: var(--spacing-lg);">
+            {{- if .Params.categories -}}
+            {{- range .Params.categories -}}
+            <a href="{{ "categories/" | relURL }}{{ . | urlize }}/" class="card-tag">{{ . }}</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)